corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -91,7 +92,8 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
     var showMenu = function (menuResponse) {
         var menuListView = menusUtils.getMenuView(menuResponse);
         var appPreview = FormplayerFrontend.currentUser.displayOptions.singleAppMode;
-        var changeFormLanguage = FormplayerFrontend.currentUser.changeFormLanguage;
+        var changeFormLanguage = toggles.toggleEnabled('CHANGE_FORM_LANGUAGE');
```

Comment:
Does this mean that `changeFormLanguage` will always be true if the FF is on? I believe the original intent of `FormplayerFrontend.currentUser.changeFormLanguage' is so that this feature can be turned off on a user setting level even if the FF is on.

Reply:
Ah interesting, the change form language is actually configured just by the toggle and then checks if the apps has additional languages it does not check by a user-by-user basis, just on a domain level. Here are the [docs](https://confluence.dimagi.com/display/saas/Change+Form+Language). I was a bit confused why the toggle check was there in the first place rather than here for that reason.

Reply:
Ah, I misinterpreted the doc where it mentions user setting. In the case, good change!